.ci/pipeline/ci.yml

```diff
@@ -57,6 +57,65 @@ jobs:
       pip install isort black[jupyter]==24.1.1
       isort --profile black --check . && black --check .
     displayName: 'Linting'
+- job: LinuxCondaRecipe
+  dependsOn: Lint
+  timeoutInMinutes: 120
+  strategy:
+    matrix:
+      Python3.12_Numpy1.26:
+        PYTHON_VERSION: '3.12'
+        NUMPY_VERSION: '1.26'
+      Python3.9_Numpy1.22:
+        PYTHON_VERSION: '3.9'
+        NUMPY_VERSION: '1.22'
+  pool:
+    vmImage: 'ubuntu-22.04'
+  steps:
+  - script: |
+      bash .ci/scripts/install_dpcpp.sh
+    displayName: "dpcpp installation"
+  - script: |
+      conda config --add channels conda-forge
+      conda config --set channel_priority strict
+      conda update -y -q --all
+    displayName: "Conda update"
+  - script: |
+      conda create -y -q -n build-env conda-build conda-verify
+    displayName: "Conda create"
+  - script: |
+      export DPCPPROOT=/opt/intel/oneapi/compiler/latest
+      . /usr/share/miniconda/etc/profile.d/conda.sh
+      conda activate build-env
+      conda build . --python $(PYTHON_VERSION) --numpy $(NUMPY_VERSION)
+    displayName: "Conda build"
+- job: WindowsCondaRecipe
```

Comment:
Where is the run_test.sh occur here? (sorry if its a dumb question, I don't know much about the conda building testing standard procedures) (given https://github.com/intel/scikit-learn-intelex/pull/1992/files#r1801556134)

Reply:
@icfaust https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-section
If this section exists or if there is a `run_test.[py,pl,sh,bat,r]` file in the recipe, the package is installed into a test environment after the build is finished and the tests are run there.

About dpnp/dpctl test skipping. I think it should be noted somewhere and tracked when it could be enabled.

Reply:
I added note about dpnp/dpctl.